pkg/deploy/controller/deploymentconfig/controller_test.go

```diff
@@ -401,3 +408,251 @@ func TestHandle_existingDeployments(t *testing.T) {
 		}
 	}
 }
+
+func TestFindDetails(t *testing.T) {
```

Comment:
Two important cases I didn't see:

1. no deployments exist, image stream tag trigger points to nonexistent tag, no build exists
2. no deployments exist, image stream tag trigger points to nonexistent tag, build exists which produces tag